torchinductor/lowering.py

```diff
@@ -47,6 +47,15 @@
 }
 
 
+def has_torchvision_roi_align():
+    try:
+        from torchvision.ops import roi_align  # noqa
+
+        return roi_align is not None
+    except (ImportError, ModuleNotFoundError):
+        return False
```

Comment:
I wonder if importlib can help avoid exceptions for control flow here, no strong opinions though.